routes/xref/lib/scraper.ts

```diff
@@ -197,12 +197,9 @@ async function getAllData(baseDir: string) {
   const data: SpecsJSON[] = urlFileContent.results;
 
   const specMap: Store["specmap"] = Object.create(null);
-  const specUrls = new Set<string>();
```

Comment:
this didn't seem to be used, so removing the dead code

Reply:
Ah, yes. This is remnant of Shepherd data times (used to add URL prefixes to `#hash-ids` 🏰).